docs/docs/sample-scripts/teams/list-all-channels-teams.md

```diff
@@ -0,0 +1,52 @@
+# List all channels in Microsoft Teams team in the tenant
```

Comment:
After reviewing the script, maybe a title such as  `Export all channels in a specific Microsoft Teams team to CSV` would be more appropriate?